src/lib/services/embed-url-parser-service/embed-url-parser-service.spec.ts

```diff
@@ -150,6 +176,21 @@ describe("EmbedUrlParserService", () => {
     }
   });
 
+  it("parses mousai URLs from user input correctly and only validates embed urls", () => {
+    for (const link of validMousaiURLs) {
+      expect(EmbedUrlParserService.isMousaiLink(link)).toBeTruthy();
+      const embedURL = EmbedUrlParserService.constructMousaiEmbedURL(new URL(link));
+      expect(embedURL).toEqual(jasmine.arrayContaining(validMousaiEmbedURLs));
```

Comment:
```suggestion
      expect(validMousaiEmbedURLs).toContain(embedURL);
```

this will fix the failing test. this will check if embedURL is in the array bvalidMousaiEmbedURLs